R/raw-data.R

```diff
@@ -0,0 +1,72 @@
+#' @importFrom dplyr mutate
+#' @importFrom purrr map_dfr
+#' @importFrom readr read_csv
+#' @importFrom glue glue
+#' @importFrom vroom problems
+
+
+# The package will store releases internally, and not in raw data
+# Until that happens, this will duplicate the bundle code
+
+#' @title Get available raw bundles
+#'
+#' @description If a bundle has multiple names, then the id will be duplicated across rows
+#'
+#' @return dataframe that contains a "concept_name" and a "domain" column for each available concept
```

Comment:
```suggestion
#' @return dataframe that contains a "concept_name" and a "domain" column for each available concept
#' @keywords internal
```

Make explicit that this is a function for internal use only